src/pages/patch/patchTabs/tasks/TasksTable.tsx

```diff
@@ -0,0 +1,106 @@
+import React from "react";
+import { loader } from "components/Loading/Loader";
+import { TaskResult } from "gql/queries/get-patch-tasks";
+import { InfinityTable } from "antd-table-infinity";
+import Badge from "@leafygreen-ui/badge";
+import { ColumnProps } from "antd/es/table";
+import { NetworkStatus } from "apollo-client";
+import { StyledRouterLink } from "components/styles/StyledLink";
+import {
+  SortQueryParam,
+  PatchTasksQueryParams,
+  TableOnChange
+} from "pages/types/task";
+import { useHistory, useLocation } from "react-router-dom";
+import queryString from "query-string";
+import { rowKey } from "pages/task/TestsTableCore";
+
+interface Props {
+  networkStatus: NetworkStatus;
+  data: [TaskResult];
+}
+
+const orderKeyToSortParam = {
+  ascend: SortQueryParam.Asc,
+  descend: SortQueryParam.Desc
+};
+const getSortDirFromOrder = (order: "ascend" | "descend") =>
+  orderKeyToSortParam[order];
+
+export const TasksTable: React.FC<Props> = ({ networkStatus, data }) => {
+  const { replace } = useHistory();
+  const { search, pathname } = useLocation();
+
+  const tableChangeHandler: TableOnChange<TaskResult> = (
+    ...[, , { order, columnKey }]
+  ) => {
+    replace(
+      `${pathname}?${queryString.stringify({
+        ...queryString.parse(search),
+        [PatchTasksQueryParams.SortDir]: getSortDirFromOrder(order),
+        [PatchTasksQueryParams.SortBy]: columnKey
+      })}`
+    );
+  };
+
+  return (
+    <>
```

Comment:
what's the purpose of including a fragment here?

Reply:
right now it's not necessary, but in future will be adding all of the task filters to this return statement and then it will be needed.

Reply:
oh wait, no, what i mention above is true for the `Tasks` component. I can delete this fragment.